src/txn/txn_recover.c

```diff
@@ -971,6 +971,9 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
          */
         WT_ERR(session->iface.checkpoint(&session->iface, "force=1"));
 
+    /* Initialize the connection's base write generation after rollback to stable. */
```

Comment:
I believe that the `__recovery_file_scan` call earlier in `__wt_txn_recover` also sets the base write generation.

Reply:
Fixed.